### PR TITLE
Patch: Commented unsafe checker

### PR DESCRIPTION
**Pull request by zktony**
## Describe your changes
Patch: Commented unsafe checker

### CODE REVIEW

File: primitives/orderbook/src/recovery.rs [hunk 1]
```diff
@@ -69,3 +69,16 @@ impl ObCheckpoint {
 		}
 	}
 }
+
+#[serde_as]
+#[derive(Clone, Debug, Encode, Decode, Default, serde::Serialize, serde::Deserialize, TypeInfo)]
+pub struct DeviationMap {
```
Comment (serhii-temchenko) [inline primitives/orderbook/src/recovery.rs:76]: Please add documentation

File: scripts/check_deviation.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (serhii-temchenko) [inline scripts/check_deviation.sh]: Please add `#!/bin/bash` at the beginning

File: pallets/ocex/rpc/src/lib.rs [hunk 3]
```diff
@@ -157,18 +158,27 @@ where
...
+		let json = serde_json::to_string(&deviation).map_err(runtime_error_into_rpc_err)?;
 		Ok(json)
 	}
 
 	async fn fetch_checkpoint(
 		&self,
 		at: Option<<Block as BlockT>::Hash>,
 	) -> RpcResult<ObCheckpoint> {
-		self.deny_unsafe.check_if_safe()?;
+		//self.deny_unsafe.check_if_safe()?;
```
Comment (serhii-temchenko) [inline pallets/ocex/rpc/src/lib.rs]: Why is this commented out? at least, please add some inline comments why it is commented out and not just removed.

File: pallets/ocex/src/validator.rs [hunk 4]
```diff
@@ -330,6 +338,15 @@ impl<T: Config> Pallet<T> {
...
 	}
 
+	// Process bacth
+	// Commit
+	// Process B2
+	// Commit
+
+	//Fetch checkpoint
+	//Processing It
+	//Commting
```
Comment (serhii-temchenko) [inline pallets/ocex/src/validator.rs]: What is this about?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

**Review by serhii-temchenko — COMMENTED** [review]
CI should be fixed after merge of PR #869.
Will approve after merge of that PR and re-run and success pass of the CI here.

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**